### PR TITLE
Feat add notes

### PR DESCRIPTION
**Pull request by krishanu2911**
Added 
- Add note
- edit note 
- change colour for note
- archive note
- trash note
- delete note

### CODE REVIEW

File: components/NoteForm.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    content: "",
+    date: new Date().toLocaleString(),
+    isArchived: false,
+    isPinned: false,
+    isTrashed: false,
+    label: "Work",
+    priority: "High",
+    title: "",
+  }
+  // const [userNote, setUserNote] = useState(initialNoteData);
```
Comment (sadathshariff) [inline components/NoteForm.js:17]: Remove the commented code

File: components/NoteForm.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  const { user } = useAuth();
+  const { setUserNotes , userNote, setUserNote , isEditing , setIsEditing , oldNote } = useUserData();
+  const handleChange = (e) => {
+    console.log("hello")
```
Comment (sadathshariff) [inline components/NoteForm.js:21]: Remove console logs

File: components/NoteForm.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+          <option value="Home">Home</option>
+          <option value="Chores">Chores</option>
+          <option value="Exercise">Exercise</option>
+        </select>
+        <button className=" p-4 bg-gray-900 rounded-md text-white font-bold">{isEditing ? "Edit" : "Add Note"}</button>
+      </form>
+    </div>
+  );
+}
+// const ReactQuill = dynamic(
```
Comment (sadathshariff) [inline components/NoteForm.js:92]: Here as well

**Review by sadathshariff — APPROVED** [review]
Everything looks fine, remove commented code and console logs

**Review by nakulsharma15 — APPROVED** [review]
Looks good. Just remove the commented code after it's not needed. No major changes suggested.

**Pull request merged**